### ISSUE TITLE
add Support Angular 16

### ISSUE THREAD

**Issue by vickyRathee** [aitboudad/ngx-loading-bar#202]
It doesn't work with Angular 16, 17, 18. 

Is this project dead, as the last publish was 3 years ago!

**Comment by aitboudad** [comment]
Hi, I think the last release still support the latest Angular version, did you get any issues from your side ?

**Comment by simon-kunz** [comment]
Hey
I too get the following error when compiling using Angluar 18.2.7:

` node_modules/@ngx-loading-bar/http-client/http.module.d.ts:1:22
    1 export declare class LoadingBarHttpClientModule {
                           ~~~~~~~~~~~~~~~~~~~~~~~~~~
    This likely means that the library (@ngx-loading-bar/http-client) which declares LoadingBarHttpClientModule is not compatible with Angular Ivy. Check if a newer version of the library is available, and update if so. Also consider checking with the library's authors to see if the library is expected to be compatible with Ivy.
`

**Comment by aitboudad** [comment]
Thanks, I'll check it and provide a new release at the end of this week!

**Comment by aitboudad** [comment]
This issue has been fixed as part of [v7.0.0](https://github.com/aitboudad/ngx-loading-bar/releases/tag/v7.0.0) release.
